index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Page</title>
+</head>
+<body>
+    <header id="top-navigation">
+        <div class="profile">
+            <a class="name" href="https://www.linkedin.com/in/patrick-ndung-u-251a1518b?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3B2E90C86VT4m5KKL5YVoCXg%3D%3D">Patrick Maina</a>
+        </div>
+        <nav id="side-nav">
+            <a class="fa fa-bars" href="#"></a>
+            <div class="portfolio">
+                <a href="#">Portfolio</a>
+            </div>
+            <div class="about">
+                <a href="#">About</a>
+            </div>
+            <div class="contact">
+                <a href="#">Contact</a>
+            </div>
+            <div class="mail">
+                <a href="#">
+                    <i class="fa fa-envelope-o"></i>
+                </a>
+            </div>
+
+        </nav>
+
+    </header>
+
+    <section class="header">
+        <h1>Patrick Maina</h1>
+        <h2>Software Developer</h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.
+        </p>
+        <div class="abilities">
+            <ul class="languages">
+                <li>Language</li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+
+            <ul class="frameworks">
+                <li>Frameworks</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+                
+            </ul>
+
+            <ul class="skills">
+                <li>Skills</li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
```

Comment:
- [ ] According to the design the `language` and `framework` should be on one row and skills should be on the next line so kindly fix it
- [ ] `li` tag has different color and Kinldy fix the color of Language, frameworks and skills
### your design
![his](https://user-images.githubusercontent.com/81727988/170518173-6b79efa2-ef9e-4153-89dc-db031f20916c.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/170518223-69bb8f2a-8978-4631-919c-8a7c82020014.png)

Reply:
resolved :heavy_check_mark: